app/assets/stylesheets/landing_page/sections/hero.scss

```diff
@@ -14,6 +14,13 @@
   box-shadow: inset 0 0 0 999999px rgba(0, 0, 0, 0.5); /* is there any cleaner way? */
 }
 
+.hero__content {
+  max-width: 1366px;
```

Comment:
`1366px` looks like a magic number, is it some breakpoint and could it be in a variable?